src/it/scala/com/databricks/spark/redshift/ColumnMetadataSuite.scala

```diff
@@ -47,7 +47,6 @@ class ColumnMetadataSuite extends IntegrationSuiteBase {
       }
     } finally {
       conn.prepareStatement(s"drop table if exists $tableName").executeUpdate()
-      conn.commit()
```

Comment:
I remember our offline discussion about the new auto-commit behavior. Can you mention that on the high level summary and link to the new behavior on docs?

Reply:
Yes I remember.  So, all `conn.commit()` that I removed are in Integration-testing Suites. The connector seems mostly used in tests to connect to redshift in .beforeAll() and .afterAll() calls to prepare the test data before spark-redshift tests read/write from redshift.

The connector is also used to unload from redshift to s3 in `RedshiftRelation` and to COPY to Redshift but that code is completely internal to the package. Each connection is properly `.close()` and all integration test are green.

Since the upgrade is not user-facing but handled internally to the package, I don't think we need to surface this new behaviour.